scripts/installer-lang/it.ps1

```diff
@@ -1,53 +1,47 @@
 [PSCustomObject]@{
     Welcome         = "
-╔══════════════════════════════╗
-║ Welcome to SpotX for Windows ║
-╚══════════════════════════════╝"
-    Incorrect       = "Ops! Valore sbagliato,"
-    Incorrect2      = "inserisci di nuovo"
+╔═══════════════════════════════╗
+║ Benvenuti a SpotX per Windows ║
+╚═══════════════════════════════╝"
+    Incorrect       = "Ops, un valore sbagliato,"
+    Incorrect2      = "inserisci di nuovo "
     Download        = "Errore nel download"
     Download2       = "Nuova richiesta in 5 secondi..."
     Download3       = "Nuovo errore"
-    Download4       = "Verifica le tue impostazioni di rete ed esegui di nuovo l'installazione."
+    Download4       = "Verifica le tue impostazioni di rete ed esegui di nuovo l'installazione"
     Download5       = "Scaricamento di Spotify"
-    Download6       = "Errore Curl"
     StopScript      = "Lo script è stato fermato"
-    MsSpoti         = "Trovata versione del Microsoft Store di Spotify, che non è supportata"
-    MsSpoti2        = "Disinstallare la versione Microsoft Store di Spotify [Y/N]"
+    MsSpoti         = "È stata rilevata la versione di Spotify del Microsoft Store, che non è supportata"
+    MsSpoti2        = "Disinstallare la versione del Microsoft Store di Spotify ? [Y/N]"
     MsSpoti3        = "Disinstallazione automatica di Spotify MS..."
     MsSpoti4        = "Disinstallazione di Spotify MS..."
     Prem            = "Modifica per account premium..."
     OldV            = "Trovata vecchia versione di Spotify"
-    OldV2           = "La tua versione di Spotify {0} è vecchia, è consigliato aggiornare alla versione {1}"
-    OldV3           = "Vorresti aggiornare? [Y/N]"
+    OldV2           = "La tua versione di Spotify {0} è vecchia, la versione attuale più recente è — {1}"
+    OldV3           = "Vorresti aggiornare ? [Y/N]"
     AutoUpd         = "Aggiornamento automatico alla versione consigliata"
-    DelOrOver       = "Vorresti disinstallare la versione installata {0} o sovrascriverla? Y [Disinstalla] / N [Sovrascrivi]"
+    DelOrOver       = "Rimuovere la versione attuale {0} o sovrascriverla? Y [Rimuovi] / N [Sovrascrivi]"
     DelOld          = "Disinstallando vecchia versione di Spotify..."
     NewV            = "Trovata versione di Spotify non supportata"
-    NewV2           = "La tua versione {0} di Spotify non è stata ancora testata, al momento la {1} è stabile"
-    NewV3           = "Vorresti continuare ad installare la versione {0} (possibili errori)? [Y/N]"
-    Recom           = "Vorresti installare la versione consigliata {0}? [Y/N]"
+    NewV2           = "La tua versione {0} di Spotify non è stata testata. La versione stabile di Spotify è {1}"
```

Comment:
in the second part of the sentence it is meant as `stable version for SpotX is {1}`

not for Spotify

Reply:
Oops, my bad. Fixed.